skycultures/greek_farnese/constellationship.fab

```diff
@@ -0,0 +1,54 @@
+Lep	15	24305	24786	24786	24927	24927	25606	25606	28816	28816	27288	27288	24786	24927	23685	28816	27654	27654	27072	28816	28103	28103	27517	24305	24845	24845	24873	24305	24327	24327	24244																																																						
+CMa	25	30122	32759	32759	33579	33579	34444	30122	31870	31870	33152	33152	33977	33856	33152	33152	33302	33302	33347	33347	32349	32349	32492	32492	33160	33160	33345	33345	34045	34045	35415	35415	35904	35904	35037	35037	33856 	33092	31592	31592	30324	30324	31700	31700	33077	33248	31416	31416	31125	31125	29843
+CMi	1	37279	36188																																					
+Arg	16	43409	42828	42828	42515	42515	42312	39429	39757	39757	38835	38835	38170	38170	38070	38070	35264	35264	36377	36377	38827	38827	42913	42913	45941	45941	46651	46651	44816	44816	39429	39953	30438
+Hya	19	43813	43109	43109	42313	42313	42402	42402	42799	42799	43813	43813	44659	44659	45336	45336	46776	46776	46509	46509	46390	46390	47452	47452	48356	48356	49402	49402	51069	51069	52085	52085	52943	52943	57936	57936	64962	64962	68895																																						
+Crt	7	53740	55705	55705	57283	57283	58188	58188	56633	56633	55687	55687	55282	55282	53740																																																												
+Crv	6	61359	60189	59199	59316	59316	60189	60189	59803	60189	60965	60965	61174																																																																
+Cen	30	67786	67457	67457	67153	67153	67786	68933	67457	67457	65109	65109	65936	65936	65373	65373	61622	61622	59184	59184	59449	66657	71683	68933	68862	68933	68245	68245	68282	68282	68002	68862	71352	71352	73334	68002	67234	67234	66657	66657	66821	66821	68702	66657	60059	60059	61084	61084	62434	60059	59747	59747	60718	60059	59449	59449	59196	59196	56480	65109	62896																				
+Lup	13	76945	78918	78918	78384	78384	76945	78384	76297	76297	74911	74911	74395	74395	71536	74395	71860	71860	70576	71860	73273	73273	75141	75141	73493	75141	76297																																																	
+Ara	7	86092	88714	88714	85258	85258	85267	85267	83081	83081	83153	83153	85792	85792	86092																																																													
+CrA	12	92308	92382	92382	92953	92953	93542	93542	94005	94005	94160	94160	94114	94114	93825	93825	93174	93174	92989	92989	91875	91875	90887	90887	90982																																																
+PsA	17	113368	111954	111954	111138	111138	109789	109789	108661	108661	107608	107608	107380	107380	108085	108085	109285	109285	111188	111188	112948	112948	113246	113246	113234	113234	113368	109111	108543	108543	108294	108035	107344	107344	106327	
+Com	10	60742	59923	59923	59489	59489	59364	59364	59468	59468	59847	60697	60351	60351	60327	60327	60941	60941	61394	61394	60697	
+
+HaR	0 78820																																																	
+HaL 0 36188
+
+Pat 0 41262
```

Comment:
Please add new line here

Reply:
hm ... I only needed a label position and, thus, entered a single star (as it is also done for Chinese single star asterisms). There strange patterns are not real constellations.

Reply:
The problem was in the correctly reading the data